VideoApi/VideoApi.Domain/Enums/VirtualCourtRoomType.cs

```diff
@@ -5,6 +5,7 @@ public enum VirtualCourtRoomType
         JudgeJOH = 1,
         Participant = 2,
         WaitingRoom = 3,
-        Civilian = 4
+        Civilian = 4,
```

Comment:
Room type is still interpreter. Participant_type ( only relevant when we call Kinly api> create room) is witness or civillian